docs/reference/api.rst

```diff
@@ -426,6 +426,10 @@ The following attributes allow more direct access to record creation.
 
     See `../how-to/use-soft-records` for a full example of its usage.
 
+.. function:: ClearRecords()
+    This can be used to remove all created records. This means the same record
+    names can be re-used for different record types.
+
```

Comment:
I would expand on this a little, just to say:

> This cannot be used once the record database has been loaded, and is only designed to be used to help with testing.

Reply:
Done in c422b3404a2b25435c41a4f4c82da8dd9731ca4b